### PR TITLE
test(ci): NODE 6 continuous integration

### PR DESCRIPTION
**Pull request by jkuri**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/angular/angular-cli.git"
   },
   "engines": {
-    "node": ">= 4.1.0"
+    "node": ">= 5.0.0"
```
Comment (Brocco) [inline package.json]: I don't think we want to exclude anyone still on node 4 even though were not running tests for it.

Comment (jkuri) [inline package.json]: @hansl said that we don't support node 4 anymore.

Comment (Brocco) [inline package.json]: then leave it as is

Comment (jkuri) [inline package.json]: okay

File: package.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
     "exists-sync": "0.0.3",
     "minimatch": "^3.0.0",
     "mocha": "^2.4.5",
-    "mock-fs": "^3.8.0",
+    "mock-fs": "^3.9.0",
```
Comment (filipesilva) [inline package.json]: Why does this dependency need updating?

File: package.json (hunk 1 above)
Comment (hansl) [inline package.json]: I thought you were asking about if we still tested on node 4. It's fine to keep node 4.1, unless there's a technical problem with this PR.

File: package.json (hunk 2 above)
Comment (filipesilva) [inline package.json]: Mike has told me it was to support node 6.

**Comment by filipesilva** [comment]
Lgtm if it passes tests. Seems to have an issue with OSX.

File: package.json (hunk 2 above)
Comment (jkuri) [inline package.json]: This was the update I was waiting for, before it was not support node6.

File: package.json (hunk 1 above)
Comment (jkuri) [inline package.json]: Reverted.

**Comment by jkuri** [comment]
Closing. @hansl will take over of this.

**Pull request closed without merging**